pyanaconda/modules/network/network.py

```diff
@@ -64,7 +64,7 @@ def __init__(self):
         self._firewall_module = FirewallModule()
 
         self.hostname_changed = Signal()
-        self._hostname = ""
+        self._hostname = conf.network.hostname
```

Comment:
What does this do for other profiles? I assume nothing, since they have nothing set in hostname, because it is new?

Reply:
Please see my comment below.